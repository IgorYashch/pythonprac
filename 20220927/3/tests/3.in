1,2,3,4,5
6,5,4,3,1
3,4,2,3,2
1,2,3,4,5
8,7,6,9,0
9,4,5,6,3
2,4,6,4,3
8,9,7,6,5
1,2,3,5,4
2,3,4,0,9

