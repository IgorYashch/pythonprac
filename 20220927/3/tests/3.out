51,62,70,52,85
101,89,101,95,74
58,58,70,61,61
51,62,70,52,85
143,132,151,157,111
